tests/crypto/test_signing_power.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from nkms.keystore import keypairs
+from nkms.crypto import api as API
+from nkms.crypto.powers import SigningPower
+
+
+class TestSigningPower(unittest.TestCase):
+    def setUp(self):
+        self.keypair = keypairs.SigningKeypair()
+        self.power = SigningPower(self.keypair)
+
+    def test_signing(self):
+        msghash = API.keccak_digest(b'hello world!')
+
+        sig = self.power.sign(msghash)
+        self.assertEqual(bytes, type(sig))
+        self.assertEqual(65, len(sig))
```

Comment:
Why 65?  Does it make more sense to make this a constant?

Reply:
`v` is a version byte (usually 1 in Ethereum, iirc). `r` and `s` are each 32 bytes.
Thus, 65 bytes.

Could be constant something like `SIGNATURE_SIZE = 65`

Reply:
We have 32 hardcoded in many places, I didn't really think this would be much different.

Reply:
I think let's have nicename constants for 32, 64, and 65 for these three scenarios - it will make the tests much easier to skim.

Reply:
I think, it makes sense to replace `32` or whatever we have in the code with constants also, but that's in a totally separate PR. This one is good to merge!

Reply:
I agree, as well. We can just use `nkms.crypto.constants` and `nkms.keystore.constants`. But, yes, that would be for another PR.

Reply:
OK then, let's merge.